src/components/ThankYouPage/ThankYouPage.jsx

```diff
@@ -44,6 +45,15 @@ const ThankYouPage = ({ onSuccess }) => {
     return () => clearTimeout(timer);
   }, []);
 
+  const isHostedMSSDK = window.location.origin === HOSTED_COMPONENTS_DOMAIN;
```

Comment:
I'd move all of these outside of the component. There's no need to redefine everything on each render.
I'm sure about `isHostedMSSDK` and `handleTopNavigate`.
I just wanted to make sure. I don't expect this to happen, but it's better to be double-sure.
Is it possible that the value of local storage's `CLEENG_MY_ACCOUNT_URL` variable changes during runtime? If yes, let's leave it here, but if not - let's also move it outside of the component

Reply:
I will move `isHostedMSSDK` and `handleTopNavigate` to new util file `hostedComponentsHelper.ts`.
When it comes to `CLEENG_MY_ACCOUNT_URL`, I will leave it in component file but move outside component